scripts/azure-pipelines.yml

```diff
@@ -52,6 +52,8 @@ jobs:
 - job: ubuntu_1604_debug_x64
   pool:
     name: Hosted Ubuntu 1604
+  variables:
+    RuntimeIdentifier: ubuntu.16.04-x64
```

Comment:
The `Directory.Build.targets` is setting the default `RuntimeIdentifier` to `NETCoreSdkRuntimeIdentifier`. However, it appears that defaults to `linux-x64` and not the same RID as is returned from `dotnet --info` (which would be `ubuntu.16.04-x64` on these machines).

This will workaround the issue and allow CI to work until an appropriate fix can be resolved (and I've started a thread to try and determine what can be done here).